Products/ZenUI3/browser/resources/css/zen-cse.css

```diff
@@ -2429,6 +2429,15 @@ BYE BYE BORDERS
         padding-right: 20px;
     }
 
+    .z-cse .x-boundlist-scrollable .x-boundlist-item {
+        display: table-row;
+        text-indent: 0;
+    }
+
+    .z-cse .x-boundlist-scrollable .x-boundlist-item-inner {
+        padding: 2px 8px;
+    }
+
```

Comment:
Dealing with `background-color` issue when scrolling:

![hover-issue](https://user-images.githubusercontent.com/37290431/51847512-82b1dc80-2324-11e9-8058-e30012edcc88.gif)

Reply:
With this workaround the behaviour is following:

![hover-issue-fixed](https://user-images.githubusercontent.com/37290431/51847506-7f1e5580-2324-11e9-99ea-f6377ffb749e.gif)